static/js/services/enketo.js

```diff
@@ -86,22 +86,24 @@ angular.module('inboxServices').service('Enketo',
       });
     };
 
-    var translateXml = function(text, title, language) {
+    var translateXml = function(text, language, title) {
```

Comment:
It's not exposed outside of this server and I updated the only other caller, so should be good.